tests/libc/enc/CMakeLists.txt

```diff
@@ -1,61 +1,46 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# Create a binary for each testcase listed in ../tests.supported
-# These tests generate a large number of warnings, some of them specific to compiler versions.
-# Disabling warning-as-errors for the specific warnings is painful across multiple different compilers.
-# It was agreed to, that it is best to just disable warnings-as-errors for these tests specifically.
-
 include(oeedl_file)
-include(${CMAKE_CURRENT_LIST_DIR}/../../../cmake/get_testcase_name.cmake)
+include(add_enclave_executable)
+
+# Include the libc tests wrappers:
+include(../tests.cmake)
+
+# Generate wrappers for libc tests.
+execute_process(
```

Comment:
We might hit a hiccup here where this either caches the generated libc tests, or doesn't cache at all and rebuilds libc tests each time.

But, let's not block the PR on this. I'll investigate once it's in 😄